clients/web/src/templates/body/userList.pug

```diff
@@ -7,6 +7,9 @@
     button.g-user-create-button.btn.btn-sm.btn-default
       i.icon-plus-squared
       |  Create user
+    i.icon-male
+      span.badge
+        | #{getUsersCount}
```

Comment:
I think if we do that, it will be to heavy on the page, with the sort by, amd the create user button it's already to much,
What do you think about that ? @zachmullen  @mgrauer